apps/site/src/app/schedule/DayTwo/DayTwo.module.scss

```diff
@@ -0,0 +1,75 @@
+@use "bootstrap-utils" as bootstrap;
+
+.mapContainer {
+	width: 100vw;
+	position: relative;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: auto;
+}
+
+.map2 {
+	position: absolute;
+	width: 90%;
+	min-width: 1400px;
+	height: auto;
+}
+
+.title {
+	position: absolute;
+	top: -60px;
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+.mapPath {
+	position: absolute;
+}
+
+.pathImage {
+	height: 100%;
+	width: 100%;
+	@include bootstrap.media-breakpoint-down(md) {
+		display: none;
+	}
+}
+
+.mobilePath {
+	width: 100%;
+	height: 2690px;
+	@include bootstrap.media-breakpoint-up(md) {
+		display: none;
+	}
+}
+
+@media screen and (min-width: 1601px) {
```

Comment:
suggestion (repeating): these media queries could be replaced with the bootstrap media breakpoints.